mechbayes/models/SEIRD_renewal.py

```diff
@@ -216,14 +216,14 @@ def __call__(self,
         if confirmed is None:
             confirmed0, confirmed = (None, None)
         else:
-            confirmed0 = confirmed[0]  # this is cumulative number of cases by start date
-            confirmed = clean_daily_obs(onp.diff(confirmed))
-            
+            confirmed0 = confirmed[0]  # this is incidence number of cases by start date
+            confirmed = clean_daily_obs(confirmed[1:])
+
         if death is None:
             death0, death = (None, None)
         else: 
             death0 = death[0]
-            death = clean_daily_obs(onp.diff(death))
+            death = clean_daily_obs(death[1:])
         
```

Comment:
I have a similar comment to the one in `SEIRD.py` --- how does this affect seeding and cumulative forecasts?